@commitlint/config-lerna-scopes/index.js

```diff
@@ -16,8 +17,21 @@ function getPackages(context) {
 		.then(() => {
 			const ctx = context || {};
 			const cwd = ctx.cwd || process.cwd();
-			const lernaVersion = getLernaVersion(cwd);
 
+			const {workspaces} = require(Path.join(cwd, 'package.json'));
+			if (Array.isArray(workspaces) && workspaces.length) {
+				// use yarn workspaces
+				return Globby(
+					workspaces.map((ws) =>
+						Path.join(ws.replace(/\//, Path.sep), 'package.json')
```

Comment:
```suggestion
						Path.posix.join(ws, 'package.json')
```

From the Globby doc:
> Note that glob patterns can only contain forward-slashes, not backward-slashes, so if you want to construct a glob pattern from path components, you need to use path.posix.join() instead of path.join().